### PR TITLE
Update Markers to be built with Gradle

### PR DESCRIPTION
**Pull request by chrisbanes**
A blast from the past. I needed a drawing app example for my next gesture navigation blog post, and Markers works perfectly.

### CODE REVIEW

**Comment by dsandler** [comment]
Is there a version of this without reformatting all the Java? I'm weird about wanting `git annotate` to work.

**Comment by chrisbanes** [comment]
Sure, updated

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 29
+
+    defaultConfig {
+        applicationId "org.dsandler.apps.markers"
+        minSdkVersion 14
+        targetSdkVersion 29
```
Comment (digitalcircuit) [inline app/build.gradle:9]: To jump [`targetSdkVersion` from `19` to `29`](https://developer.android.com/distribute/best-practices/develop/target-sdk ), looking through the changes it doesn't *seem* like Markers was modified to make use of run-time permissions and scoped storage?

My experience with the current version has Markers saving directly to a location on internal storage.  (I haven't actually tried compiling and running; I may be mistaken.)

Regardless, thanks for the effort to help revive this app!

Comment (chrisbanes) [reply app/build.gradle:9]: Yeah you're right, -2 this change for now.

**Comment by dsandler** [comment]
Oh yeah I also totally forgot that I already did this in [the wear branch](https://github.com/dsandler/markers/tree/wear). Sorry, it's … been a while.

**Pull request closed without merging**